coderadar-server/coderadar-core/src/main/java/org/wickedsource/coderadar/core/rest/dates/Day.java

```diff
@@ -1,5 +1,10 @@
 package org.wickedsource.coderadar.core.rest.dates;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
```

Comment:
instead of `@Getter` and `@AllArgsConstructor`, we can just as well use `@Value` and then even remove the `hashCode` and `equals` methods.